ivy/functional/ivy/device.py

```diff
@@ -689,15 +689,16 @@ def gpu_is_available() -> bool:
 
     Returns
     -------
-    ret
-        Boolean, as to whether a gpu is available.
+    ret: bool
```

Comment:
The type hint here is unnecessary here and to be consistent with other typehints in this document, please revert this change.